mobile-app/app/api/persistence/notifiction_storage.ts

```diff
@@ -0,0 +1,52 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+
+const KEY = 'WALLET.NOTIFICATION'
+
+export enum NotificationType {
+  TRANSACTION = 'TRANSACTION',
+}
+
+export interface AppNotificationTypesI {
+  type: NotificationType
+  value: boolean
+  displayName: string
+}
+
+function getAppNotificationTypes (): Array<{
+  type: NotificationType
+  displayName: string
+  dafaultValue: boolean
+}> {
+  return [
+    {
+      type: NotificationType.TRANSACTION,
+      displayName: 'Transactions',
+      dafaultValue: true
+    }
+  ]
+}
+
+function notificationParser (preferances: AppNotificationTypesI[]): AppNotificationTypesI[] {
+  const notificationsTypes = getAppNotificationTypes()
+  return notificationsTypes.map(({ type, displayName, dafaultValue }) => {
+    const savedType = preferances.find((each) => {
```

Comment:
Can we just save the preference type? So we don't need to do double loop (map and find).